docker-compose.yml

```diff
@@ -30,11 +30,21 @@ services:
     command: ["nuodocker", "start", "te", "--db-name", "hockey", "--server-id", "nuoadmin1",
           "--servers-ready-timeout", "60"]
   influxdb:
-    image: influxdb:1.8
+    image: influxdb:2.7.1
     ports:
       # The API for InfluxDB is served on port 8086
       - "8086:8086"
       - "8082:8082"
+    environment:
+      - DOCKER_INFLUXDB_INIT_MODE
+      - DOCKER_INFLUXDB_INIT_USERNAME
+      - DOCKER_INFLUXDB_INIT_PASSWORD
+      - DOCKER_INFLUXDB_INIT_ORG
+      - DOCKER_INFLUXDB_INIT_RETENTION
+      - DOCKER_INFLUXDB_INIT_BUCKET
+      - DOCKER_INFLUXDB_INIT_ADMIN_TOKEN
```

Comment:
Where are all these set? If it's required for the user to set them, this should be mentioned in the [quickstart instructions](https://github.com/nuodb/nuodb-collector/blob/master/README.md#quick-start-with-docker-compose).

I would just set the variables here to the default values and use [docker compose secret](https://docs.docker.com/compose/use-secrets/) for the token which should be pre-generated by the user.